profiles/aparna2071.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/54634804/180271682-7083563f-cb70-4b00-b562-158849d7bea5.png",
+  "issueId": 1249 ,
```

Comment:
There's an extra space in here.
```suggestion
  "issueId": 1249,
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Thanks @Panquesito7